### PR TITLE
Convert to proper Symfony requests

### PR DESCRIPTION
**Pull request by Nyholm**
FYI @GrahamCampbell

### CODE REVIEW

File: src/bref/src/LaravelHttpHandler.php [hunk 1]
```diff
@@ -25,42 +24,11 @@ public function __construct(Kernel $kernel)
...
-            'REQUEST_TIME_FLOAT' => microtime(true),
-            'REQUEST_URI' => $event->getUri(),
-        ], fn ($value) => null !== $value);
-
-        foreach ($event->getHeaders() as $name => $values) {
-            $server['HTTP_'.strtoupper($name)] = $values[0];
-        }
-
-        // TODO convert request better
-        $request = Request::create(
```
Comment (Nyholm) [inline src/bref/src/LaravelHttpHandler.php:50]: Is this an issue that we are now using `Symfony\Component\HttpFoundation\Request` to the Laravel kernel?

Comment (Nyholm) [reply src/bref/src/LaravelHttpHandler.php:50]: The interface says that it is accepted.

Comment (GrahamCampbell) [reply src/bref/src/LaravelHttpHandler.php:50]: > The interface says that it is accepted.

Don't trust what Laravel phpdoc says. :trollface:

Comment (GrahamCampbell) [reply src/bref/src/LaravelHttpHandler.php:50]: You should do:

```php
$request = Illuminate\Http\Request::createFromBase(
    SymfonyRequestBridge::convertRequest($event, $context)
);
```

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/bref/src/SymfonyRequestBridge.php [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @internal
+ */
+class SymfonyRequestBridge
+{
+    public static function convertRequest(HttpRequestEvent $event, Context $context): Request
+    {
+        Request::setTrustedProxies(['127.0.0.1'], Request::HEADER_X_FORWARDED_FOR | Request::HEADER_X_FORWARDED_HOST | Request::HEADER_X_FORWARDED_PORT | Request::HEADER_X_FORWARDED_PROTO);
```
Comment (GrahamCampbell) [inline src/bref/src/SymfonyRequestBridge.php:25]: Why is this needed? Can't we just set the real headers, and not have the forwarded for headers making an appearance?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/bref/src/SymfonyRequestBridge.php (hunk 2 above)
Comment (Nyholm) [reply src/bref/src/SymfonyRequestBridge.php:25]: Yeah, We could. But Im not sure what all sideeffects would be. Think if multiple proxies etc. It is probably fine, but I wasn't 100% sure so I left it for a future update.
